src/components/ImageUploader.js

```diff
@@ -204,8 +256,9 @@ class ImageUploader extends React.Component {
       case 0:
         return (
           <ImageSelector
-            label={this.vars.imageLabel}
-            imageName={this.state.imageName}
+            // label={this.vars.imageLabel}
```

Comment:
Why comment this?

Reply:
I do not need label any more because I don't display all of the pictures anymore. I just display the image names, whose information is already stored in imageName.

Here is my new UI

![image](https://user-images.githubusercontent.com/10831251/36331321-613b3b72-133b-11e8-8faf-bc2183922e68.png)

Reply:
Can you add a condition on board image to disappear in case there are more than one image selected?